src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -363,15 +374,28 @@ trait BronzeTransforms extends SparkSessionWrapper {
         .withColumn("date", 'time.cast("date"))
         .select('category, 'version, 'timestamp, 'date, 'properties, 'identity.alias("userIdentity"), 'organization_id)
         .withColumn("userIdentity", structFromJson(spark, schemaBuilders, "userIdentity"))
-        .selectExpr("*", "properties.*").drop("properties")
+        .selectExpr(
+          "*",
+          "properties.actionName",
+          "properties.logId",
+          "properties.requestId",
+          "properties.requestParams",
+          "properties.response",
+          "properties.serviceName",
+          "properties.sessionId",
+          "properties.sourceIPAddress",
+          "properties.userAgent")
+        .drop("properties")
```

Comment:
Can we add these column list in Schema.scala and then use this list instead of passing the hardcoded column names ?